src/docs/command-line.dox

```diff
@@ -204,9 +204,9 @@ a table, creating the table if it does not yet exist.  The data should
 be the format produced by the \c dump command; see @ref dump_formats for
 details.
 
-By default, if the table already exists, data in the table will be
-overwritten by the new data (use the \c -n option to make an attempt to
-overwrite existing data return an error).
+By default, if the table already exists, key/value pairs in the table
+will be overwritten by new data with matching keys (use the \c -n option
+to make an attempt to overwrite existing data return an error).
```

Comment:
I would suggest even being more detailed/blunt and saying something like "this is, if the new data has fewer keys, existing keys will not be removed" or something like that.

Reply:
I like being more explicit.  I modified your wording a bit.